keras_nlp/models/bert.py

```diff
@@ -391,17 +421,36 @@ def BertBase(weights=None, vocabulary_size=None, name=None, trainable=True):
 
     # TODO(jbischof): consider changing format from `h5` to
```

Comment:
Just move this whole todo into the helper function next to the `get_file` call. `load_weights` already supports tf checkpoints, so we really would change this inside the helper not here I believe.